frontend/test/app/components/EmailDialog-test.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { shallow } from 'enzyme';
+
+import EmailDialog from '../../../src/app/components/EmailDialog';
+
+describe('app/components/EmailDialog', () => {
+
+  const mockClickEvent = {
+    preventDefault() {},
+    stopPropagation() {}
+  };
+
+  beforeEach(function() {
+    global.window = {
```

Comment:
Hmm this window / ga stuff is coming up a lot, I should refactor it into a shared util if I can